api/worker.ml

```diff
@@ -39,6 +39,7 @@ module Solve_request = struct
     pinned_pkgs : (string * string) list;
         (** Name and contents of other pinned opam files. *)
     platforms : (string * Vars.t) list;  (** Possible build platforms, by ID. *)
+    prefer_oldest : bool;  (** Solve for the oldest possible versions instead of newest. *)
```

Comment:
I think `lower_bound` sounds more for the request than `prefer_oldest` even if it is the same parameter name of `Git_context.create`.